test/integration/integration-test.js

```diff
@@ -80,8 +86,16 @@ function teardown(done) {
 describe.skip('integration tests', function() {
   const instructions = {maxLedgerVersionOffset: 100};
   this.timeout(TIMEOUT);
+  const transactions = [];
 
+  // beforeEach(setup);
   beforeEach(setup);
+  beforeEach(function() {
+    if (!this.startLedgerVersion) {
```

Comment:
How does this work? Is `this.startLedgerVersion` ever set at this point? Can we move this into the `setup` method so that `beforeEach` is only called once?